js/sites.js

```diff
@@ -72,11 +72,12 @@
         aIndex = this.cell.index;
       }
 
-      gNewTab.sendToBrowser("NewTab:PinLink", {
+      gNewTab.sendToBrowser("NewTab:GetUpdatePages", {
```

Comment:
You might want to include an ID so that this page doesn't update itself.
This will cause the browser to send a `NewTab:UpdatePages` message to all pages, including this one, causing a flicker.